.circleci/config.yml

```diff
@@ -136,18 +136,6 @@ jobs:
           name: Deploy Contracts
           working_directory: ~/protocol/core
           command: $(npm bin)/truffle migrate --reset --network test
-      - restore_cache:
```

Comment:
Just to understand why did you change this?

Reply:
Yeah, this should've been changed before, when we moved the voter-dapp to lerna. It should be relying on the `npx lerna bootstrap` called in the `checkout_and_install` step to install all of its dependencies. However, the local continued to work until it needed to install other sub-packages (like common).